### PR TITLE
Document the available annotations

### PR DESCRIPTION
**Pull request by DewaldV**
Update the docs regarding the annotation for yggdrasil.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,8 +50,23 @@ Your ingress set up then looks like this:
 
 Where the envoy nodes are loadbalancing between each cluster for a given ingress.
 
-### Health Check
-Yggdrasil always configures a path on your Envoy nodes at `/yggdrasil/status`, this can be used to health check your envoy nodes, it will only return 200 if your nodes have started and been configured by Yggdrasil.
```
Comment (Joseph-Irving) [inline README.md]: why have you deleted this?

Comment (DewaldV) [reply README.md]: My bad, I didn't read the text and assumed it was about the health check annotation (that we used to always configure). Will fix.

**Comment by Joseph-Irving** [comment]
looks good, maybe including an example ingress would be useful?

**Comment by DewaldV** [comment]
Added that example you were looking for. 👍

**Pull request merged**